Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 664, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
kins, pots, Ps, hist = simulate()

T = np.mean(kins * 2 / (3 * N))
P = 1 - np.mean(Ps) / (3 * N * T) - 16 * np.pi * rho / (3 * T * L_box**3)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
[0;32m/tmp/ipykernel_2408/1421777266.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mkins[0m[0;34m,[0m [0mpots[0m[0;34m,[0m [0mPs[0m[0;34m,[0m [0mhist[0m [0;34m=[0m [0msimulate[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0;34m[0m[0m
[1;32m      3[0m [0mT[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0mmean[0m[0;34m([0m[0mkins[0m [0;34m*[0m [0;36m2[0m [0;34m/[0m [0;34m([0m[0;36m3[0m [0;34m*[0m [0mN[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0mP[0m [0;34m=[0m [0;36m1[0m [0;34m-[0m [0mnp[0m[0;34m.[0m[0mmean[0m[0;34m([0m[0mPs[0m[0;34m)[0m [0;34m/[0m [0;34m([0m[0;36m3[0m [0;34m*[0m [0mN[0m [0;34m*[0m [0mT[0m[0;34m)[0m [0;34m-[0m [0;36m16[0m [0;34m*[0m [0mnp[0m[0;34m.[0m[0mpi[0m [0;34m*[0m [0mrho[0m [0;34m/[0m [0;34m([0m[0;36m3[0m [0;34m*[0m [0mT[0m [0;34m*[0m [0mL_box[0m[0;34m**[0m[0;36m3[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/tmp/ipykernel_2408/950665432.py[0m in [0;36msimulate[0;34m(freq_T, freq_out, nsteps)[0m
[1;32m      6[0m     [0mvel[0m [0;34m=[0m [0mIC_vel[0m[0;34m([0m[0mN[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      7[0m [0;34m[0m[0m
[0;32m----> 8[0;31m     [0mF[0m [0;34m=[0m [0mfind_force[0m[0;34m([0m[0mpos[0m[0;34m)[0m[0;34m[[0m[0;36m0[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      9[0m [0;34m[0m[0m
[1;32m     10[0m     [0;32mfor[0m [0mt[0m [0;32min[0m [0mrange[0m[0;34m([0m[0;36m10000[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;31mTypeError[0m: find_force() missing 1 required positional argument: 'L_box'
TypeError: find_force() missing 1 required positional argument: 'L_box'

